cypress/e2e/hub/collections.cy.ts

```diff
@@ -57,49 +58,106 @@ describe('Collections- List View', () => {
     cy.galaxykit('collection -h');
   });
 
-  it.skip('user can delete a collection using the list toolbar', () => {});
+  it('user can delete a collection using the list toolbar', () => {
+    const namespace = 'hub_e2e_namespace_' + randomString(3, undefined, { isLowercase: true });
+    const collection =
+      'hub_e2e_collection_' + randomString(5, undefined, { isLowercase: true }).replace(/\d/g, '');
+    cy.createNamespace(namespace);
+    cy.uploadCollection(collection, namespace);
+    cy.approveCollection(collection, namespace, '1.0.0');
```

Comment:
@skochay330  Could we get the version from the uploadCollection? Not sure if it is always `1.0.0`.

Reply:
If a version is not given during time of galaxykit `uploadCollection` command, it defaults to 1.0.0

Reply:
and our current cypress `uploadCollection` command does not accept a `version` parameter so we should be good

Reply:
ok, I was thinking about something
 ```
cy.uploadCollection(collection, namespace).then((result) => {
``` 
result has the version